terrain: "Flat"
# Options: Flat, Step, Rough, SineLowFreq, SineHighFreq, SlopePlat, Gap, TwoSlope, MultipleSlopes, TwoStep, FiveSteps
run_derivative_test: false
plot_trajectory: false
play_initialization: false
shift_initialization: false
init_state_from_file: true  # 'true' if initial_pose for base is read from this file, 'false' if it is set by the current anymal state

constrain_initial_y_base: true
constrain_final_y_base:   false 
constrain_final_z_base:   false
constrain_final_ee_pos:   false
constrain_final_base_orientation: false 

offset_reference_height: false
offset_height: -0.06

set_wheels_motion_cost: false
set_base_pitch_cost: false

SlopePlat:
  initial_pose:
    x: 0.0 
    y: 0.0
    z: 0.42
   
  final_pose: 
    x: 1.8
    y: 0.0
    z: 0.65

  total_time: 4.0
  use_wheels_motion_constraint: false
  traction_limit: 10000.0 # 10000 for 65

Flat:
  initial_pose:
    x: 0.0
    y: 0.0
    z: 0.48
   
  final_pose: 
    x: 1.0 #2.0
    y: 0.0
    z: 0.48

  total_time: 1.0 #6.0
  use_wheels_motion_constraint: false
  traction_limit: 400 #30.0 

Step:

  initial_pose:
    x: 0.0
    y: 0.0
    z: 0.45
   
  final_pose: 
    x: 2.5
    y: 0.0
    z: 0.22

  total_time: 5.0
  use_wheels_motion_constraint: false
  traction_limit: 300.0

Rough:
  initial_pose:
    x: 0.0
    y: 0.0
    z: 0.5
   
  final_pose: 
    x: 4.0
    y: 0.0
    z: 0.95

  total_time: 3.5
  use_wheels_motion_constraint: false
  traction_limit: 30.0

FiveSteps:
  initial_pose:
    x: 0.0
    y: 0.0
    z: 0.5
   
  final_pose: 
    x: 3.6
    y: 0.0
    z: 1.45

  total_time: 7.2 #7.2 # 8.2, 8.6, 9.6
  use_wheels_motion_constraint: false
  traction_limit: 400.0

SineHighFreq:
  initial_pose:
    x: 0.0
    y: 0.0
    z: 0.5
   
  final_pose: 
    x: 4.0
    y: 0.0
    z: 0.5

  total_time: 4.0
  use_wheels_motion_constraint: false
  traction_limit: 30.0

SineLowFreq:
  initial_pose:
    x: 0.0
    y: 0.0
    z: 0.6
   
  final_pose: 
    x: 8.0
    y: 0.0
    z: 0.6

  total_time: 8.0
  use_wheels_motion_constraint: false
  traction_limit: 30.0

Slope:
  initial_pose:
    x: 0.0
    y: 0.0
    z: 0.5
   
  final_pose: 
    x: 3.5
    y: 0.0
    z: 0.4

  total_time: 4.5
  use_wheels_motion_constraint: false
  traction_limit: 400.0

Gap:
  initial_pose:
    x: 0.0
    y: 0.0
    z: 0.95
   
  final_pose: 
    x: 3.5
    y: 0.0
    z: 0.95

  total_time: 3.0
  use_wheels_motion_constraint: false
  traction_limit: 400.0

TwoSlope:
  initial_pose:
    x: 0.0
    y: 0.0
    z: 0.5
   
  final_pose: 
    x: 3.0
    y: 0.0
    z: 0.5

  total_time: 3.5 #4.0
  use_wheels_motion_constraint: false
  traction_limit: 300.0

MultipleSlopes:
  initial_pose:
    x: 0.0
    y: 0.0
    z: 0.50
   
  final_pose: 
    x: 1.7
    y: 0.0
    z: 0.50

  total_time: 3.5
  use_wheels_motion_constraint: false
  traction_limit: 400.0

TwoStep:
  initial_pose:
    x: 0.0
    y: 0.0
    z: 0.5
   
  final_pose: 
    x: 2.2
    y: 0.0
    z: 0.8

  total_time: 3.6 #3.6 #4.1 #4.0
  use_wheels_motion_constraint: false
  traction_limit: 400.0 #300.0

use_non_holonomic_constraint: false







